blocks.py

```diff
@@ -18,4 +18,24 @@ def gyro_straight(tank, gyro, millies, target):
         subtract = gyro.angle() - target  # subtracts color sensors
         multiply = subtract * SHARPNESS # multiplies the difference by 0.7
         tank.drive(SPEED, multiply) # follows the line
+        
+        
+def stop_on_black(left_cs, right_cs):
+        if left_cs.reflection() <= 14:
```

Comment:
We should check both sensors here:

```suggestion
        if left_cs.reflection() <= 14 and right_cs.reflection() <= 14:
```
we should also make 14 a constant.